tiago_clf_launch/config/tiago_motions.yaml

```diff
@@ -536,6 +536,28 @@ play_motion:
       - positions: [0.09, 1.02, -3.01, 0.74, -1.54, 1.29, -0.20, 0.15]
         time_from_start: 2.0
 
+    transport_over_head_slightly_forward:
+      joints: [arm_1_joint, arm_2_joint, arm_3_joint, arm_4_joint, arm_5_joint, arm_6_joint, arm_7_joint, 'torso_lift_joint']
+      meta: {description: transport_over_head, name: transport over head, usage: demo}
```

Comment:
Meta is defined twice.
Name already used make sure name is unique and not too long as they are used in the webgui